setup.py

```diff
@@ -80,9 +80,11 @@
       test_suite='plugin_tests',
       tests_require=test_requirements,
       ext_modules = cythonize(Extension(
-           "histomicstk.segmentation.label.isbf",
-           sources=["histomicstk/segmentation/label/isbf.pyx",
-                    "histomicstk/segmentation/label/isbfcpp.cpp"],
+           "histomicstk.segmentation.label.trace_object_boundary",
+           sources=[
+                "histomicstk/segmentation/label/trace_object_boundary.pyx",
+                "histomicstk/segmentation/label/trace_boundary_cpp.cpp"
+           ],
```

Comment:
There seems to be a way to specify sources in the pyx file instead of setup.py. See [here](http://docs.cython.org/en/latest/src/userguide/wrapping_CPlusPlus.html#specify-c-language-in-setup-py). It will make setup.py much more cleaner.